datasets/geonet.yaml

```diff
@@ -0,0 +1,38 @@
+Name: GeoNet Aotearoa New Zealand Data
+Description: GeoNet provides geological hazard information for Aotearoa New Zealand. This dataset contains data and products recorded by the GeoNet sensor network. The dataset currently include GNSS data and additional datasets will be added in the near future.
+  GNSS (Global Navigation Satellite System) data include raw data in proprietary and Receiver Independent Exchange Format (RINEX) and local tie-in survey conducted during equipment changes '(https://www.geonet.org.nz/data/types/geodetic)'.
+Documentation: https://github.com/GeoNet/data-tutorials/tree/main/AWS_Open_Data

```

Comment:
Just wanted to note that the Documentation link to your page isn't live yet. You might consider waiting to push the dataset yaml which will then be public.

Reply:
thanks @jeffejefe , yes I'm aware but it should be published soon, I'll ping my reviewers and hopefully the link will be active within a couple of days

Reply:
great thanks!

Reply:
this is now fixed, and link can be accessed